doc/changes/changes_3.4.1.md

```diff
@@ -0,0 +1,15 @@
+# Exasol Test Containers 3.4.0, released 2020-12-04
```

Comment:
```suggestion
# Exasol Test Containers 3.4.1, released 2021-XX-XX
```

Reply:
Updated to January 13th. Fingers crossed.